events/sinks/gcl/gcl.go

```diff
@@ -48,13 +49,18 @@ func (sink *gclSink) ExportEvents(eventBatch *core.EventBatch) {
 	glog.V(4).Info("Exporting events")
 	entries := make([]*gcl.LogEntry, len(eventBatch.Events))
 	for i, event := range eventBatch.Events {
+		evtJson, err := json.Marshal(event)
```

Comment:
This doesn't look right.  What kind of event is this?  Normally you'd using something like runtime.Encode.